src/core/ctf.h

```diff
@@ -112,6 +120,18 @@ class CTF {
     float               ParticleShiftGivenAzimuth(float azimuth);
     float               WavelengthGivenAccelerationVoltage(float acceleration_voltage);
 
+    inline float xi(float squared_spatial_frequency) {
+        return PIf * wavelength * squared_spatial_frequency * sample_thickness;
+    };
+
+    inline float sinc_xi(float squared_spatial_frequency) {
```

Comment:
This is okay for now, but why don't we consider having a sinc function in functions.h and we can base it off the boost implementation:

```cpp
            static T const    taylor_0_bound = tools::epsilon<T>();
            static T const    taylor_2_bound = sqrt(taylor_0_bound);
            static T const    taylor_n_bound = sqrt(taylor_2_bound);

            if    (abs(x) >= taylor_n_bound)
            {
                return(sin(x)/x);
            }
            else
            {
                // approximation by taylor series in x at 0 up to order 0
#ifdef __MWERKS__
                U<T>    result = static_cast<U<T> >(1);
#else
                U<T>    result = U<T>(1);
#endif

                if    (abs(x) >= taylor_0_bound)
                {
                    U<T>    x2 = x*x;

                    // approximation by taylor series in x at 0 up to order 2
                    result -= x2/static_cast<T>(6);

                    if    (abs(x) >= taylor_2_bound)
                    {
                        // approximation by taylor series in x at 0 up to order 4
                        result += (x2*x2)/static_cast<T>(120);
                    }
                }

                return(result);
            }
        }
```

It looks like tools::epsilon is just using std::numeric_limits<>::epsilon under the hood